Tests/UtilityTests/ProcessTests.swift

```diff
@@ -0,0 +1,92 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import XCTest
+import Basic
+import libc
+import Utility
+
+class ProcessTests: XCTestCase {
+
+    func testBasics() throws {
+        let process = Process(args: "echo", "hello")
+        try process.launch()
+        let result = try process.waitUntilExit()
+        XCTAssertEqual(try result.output.dematerialize(), "hello\n")
+        XCTAssertEqual(result.exitStatus, .terminated(code: 0))
+    }
+
+    func testPopen() throws {
+        XCTAssertEqual(try Process.popen(arguments: ["echo", "hello"]).output.dematerialize(), "hello\n")
+
+        let path = AbsolutePath(#file).parentDirectory.appending(component: "ArgumentParserTests.swift")
+        XCTAssertGreaterThan(try Process.popen(args: "cat", path.asString).output.dematerialize().characters.count, 4096)
+
+        let result = try Process.popen(args: "cat", "/bin/cat")
```

Comment:
I'm not clear what this is testing... in general I think it would be good to have a few more comments describing what the test is trying to cover.